fares-data-build-tool/fdbt-site
# Multi operator ticket type page

# Description

-   This PR makes changes to the periodType page to allow it to be used on the multi operator user flow. The name of the page/API has been changed to be more generic.

# Testing instructions

-   Pull down this branch and run the site locally. Navigate to the ticketRepresentation page (down the period ticket user flow) and validate that the page behaves as expected, and as the periodType page used to.
-   Also validate that the matching json is as expected.

# Type of change


# Checklist:
